io/GDALWriter.cpp

```diff
@@ -131,6 +129,13 @@ void GDALWriter::initialize()
             m_outputTypes |= GDALGrid::statIdw;
         else if (ts == "stdev")
             m_outputTypes |= GDALGrid::statStdDev;
+        else if (Utils::tolower(ts[0]) == 'p' && ts.size() > 1)
+        {
+            int p = std::stoi(ts.substr(1));
```

Comment:
This doesn't handle cases where you might have garbage after the integer value or locale. See Utils::from_string()